device-types/Calient/s320.yaml

```diff
@@ -0,0 +1,352 @@
+---
+manufacturer: Calient
+model: s320
+slug: calient-s320
+part_number: s320
+u_height: 7
+is_full_depth: false
+front_image: true
+airflow: passive
+weight: 20.5
+weight_unit: kg
+console-ports:
+  - name: CONSOLE
+    type: rj-45
+power-ports:
+  - name: PWR-1
+    type: dc-terminal
+    maximum_draw: 20
+  - name: PWR-2
+    type: dc-terminal
+    maximum_draw: 20
+interfaces:
+  - name: ethA
+    type: 1000base-t
+    mgmt_only: true
+  - name: ethB
+    type: 1000base-t
+    mgmt_only: true
+  - {name: 1.1.1, type: other}
```

Comment:
You still have a lot here

Reply:
sorry... forgot i had that as part of this